### DISCUSSION TITLE
Airflow Login Authentication Failing

### DISCUSSION THREAD

**Discussion by 11utkarshb** [apache/airflow#32827] [General]
I have deployed airflow on cloud connected with postgress database now when I am creating a user using standalone command the user details are getting filled in ab_user in database but the password is encrypted. Can anyone please guide where to find the password to login in Airflow webserver.

**Comment by potiuk** [comment]
You should use the password you used to create the user with. It's encrypted indeed you could if you want attempt to decrypt if you have FERNET_KEY (which is symmetric key) but you need to write a Python code for it - just look at airflow's code and see how User code looks like.
